website/source/docs/vault-enterprise/replication/index.html.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: "docs"
+page_title: "Vault Enterprise HSM Support"
+sidebar_current: "docs-vault-enterprise"
+description: |-
+  Vault Enterprise has support for Replication, allowing critical data to be replicated across clusters. 
+
+---
+
+# Vault Replication
+
+## Overview
+
+Many organizations have infrastructure that spans multiple datacenters. Vault provides 
+the critical services of identity management, secrets storage, and policy management. 
+This functionality is expected to be highly available and operators would like to ensure
+a common set of policies are enforced globally, and a consistent set of secrets and keys
+are exposed to applications that need to interoperate. Vault replication enables
+redundancy across datacenters.
+
+Prior to Vault 0.7, Vault nodes could only be paired within clusters using a common HA
+storage backend. A single node is locked with a set of other nodes in a cluster to ensure
+high availability. With replication, we intend to allow Vault to allow horizontal 
+scalability between clusters a cross geographically distributed data centers.
+
+## Architecture
+
+When replicating clusters, a single cluster is designed the primary cluster. The
+primary cluster is the active cluster in use within an infrastructure, which 
+serves as the system  of record for replication.
+
+The primary cluster asynchronously replicates data to a series of remote standby
```

Comment:
This makes the standby's sound like they do nothing. I would rephrase as "The primary cluster replications in near real-time to any number of secondary clusters."